### PR TITLE
feat: Add email and phone for all companies

### LINKED ISSUE #9: More fields for company details
## Is your feature request related to a problem?

I think we should add two fields `email` and `phone` for every company to make it easy for users to contact the respective company.

## Describe the solution you'd like

Add the key-value pair for every company in the top level of the company object.

## Describe alternatives you've considered

N/A

## Additional context

Add these missing fields for every company already listed

### PR DESCRIPTION
**Pull request by TheLearneer**
## What kind of change does this PR introduce? 

- [ ] Bug Fix
- [ ] Documentation change
- [ ] Added new company
- [x] Other(s)

## Why was this change needed?

Fixes #9 

## Other information

Users could always check the company website and then find the contact details, but having them directly in the data files would reduce 1 additional step the users would have to take.

### CODE REVIEW

File: data/cotiviti-nepal.json [hunk 1]
```diff
@@ -2,7 +2,9 @@
   "name": "Cotiviti Nepal Pvt. Ltd.",
   "establishedAt": 1072894500,
   "location": "Hattisar, Kathmandu",
-  "website": "http://www.cotiviti.com.np/",
+  "website": "https://www.cotiviti.com.np/",
```
Comment (TheLearneer) [inline data/cotiviti-nepal.json:5]: note: Switched to HTTPS

File: data/fusemachines.json [hunk 2]
```diff
@@ -3,6 +3,8 @@
   "establishedAt": 1356977700,
   "location": "Devkota Sadak, Baneshwor, Kathmandu",
   "website": "https://fusemachines.com/",
+  "email": null,
```
Comment (TheLearneer) [inline data/fusemachines.json:6]: NOTE: Couldn't find any email referring to Nepal specific team

File: data/idea-breed-technology.json [hunk 3]
```diff
@@ -2,7 +2,9 @@
   "name": "Idea Breed Technology Pvt. Ltd.",
   "establishedAt": 1388517361,
   "location": "Pargati Nagar Road, New Baneshwor, Kathmandu",
-  "website": "http://www.ideabreed.net/",
+  "website": "https://www.ideabreed.net/",
```
Comment (TheLearneer) [inline data/idea-breed-technology.json:5]: NOTE: Switched to HTTPS

File: data/lanceme-up.json [hunk 4]
```diff
@@ -1,8 +1,10 @@
 {
   "name": "Lanceme Up",
   "establishedAt": 1599978360,
-  "location": "महापाल रोड, Lalitpur",
+  "location": "Mahapal Road, Lalitpur",
```
Comment (TheLearneer) [inline data/lanceme-up.json:4]: NOTE: Changed to english to be consistent

File: data/leapfrog-technology.json [hunk 5]
```diff
@@ -3,6 +3,8 @@
   "establishedAt": 1262283300,
   "location": "Charkhal Rd, Dillibazar, Kathmandu",
   "website": "https://www.lftechnology.com/",
+  "email": "hello@lftechnology.com",
+  "phone": null,
```
Comment (TheLearneer) [inline data/leapfrog-technology.json:7]: NOTE: Couldn't find any phone number at all 🤨

Comment (thenishantsapkota) [reply data/leapfrog-technology.json:7]: 01-4519766
Check this

File: data/lanceme-up.json (hunk 4 above)
Comment (thenishantsapkota) [reply data/lanceme-up.json:4]: GG

File: data/eminance-ways.json [hunk 6]
```diff
@@ -3,6 +3,8 @@
   "establishedAt": 1364307090,
   "location": "Maitidevi, Kathmandu",
   "website": "https://eminenceways.com/",
+  "email": "info@eminenceways.com",
+  "phone": "+977-014569328",
```
Comment (thenishantsapkota) [inline data/eminance-ways.json:7]: I guess the phone number format should be `+977-1` instead of `+977-01`

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

File: data/eminance-ways.json (hunk 6 above)
Comment (TheLearneer) [reply data/eminance-ways.json:7]: Alright, fixing it in all the files then 🙇

**Review by thenishantsapkota — APPROVED** [review]

**Pull request merged**